### ISSUE TITLE
AttributeError when plotting the reassembly summary plots

### ISSUE THREAD

**Issue by SamLambrechtsUGent** [bxlab/metaWRAP#124]
Dear metaWRAP team,

While running the reassemble_bins command from the tutorial (using metawrap v 1.1.3) , something went wrong while plotting the summary plots at the end:

Plotting completion data...
Traceback (most recent call last):
  File "/opt/anaconda/bin/metawrap-scripts/plot_reassembly.py", line 68, in <module>
    ax.set_axis_bgcolor('white')
AttributeError: 'AxesSubplot' object has no attribute 'set_axis_bgcolor

Google leads me to: https://github.com/scikit-learn/scikit-learn/issues/10762... But I'm not sure I can fix this myself.

**Comment by ursky** [comment]
I'm curious - have you been able to get the same bin improvement plot from the bin_refinement module, or you haven't tried?

**Comment by SamLambrechtsUGent** [comment]
I'm not 100% sure, because it's a few days ago that I ran the bin_refinement step of the tutorial, but I'm pretty sure I didn't see a "Something went wrong with.... Exiting." message. One difference is that now I'm running metawrap 1.1.3, and when I ran bin_refinement, I was using v 1.1.2

**Comment by ursky** [comment]
Did the bin_refinement produce the plot or not?

**Comment by SamLambrechtsUGent** [comment]
Yes, both the binning results and intermediate binning results png and eps files are there

**Comment by ursky** [comment]
Ok, I think I know what happened. The plotting script uses the option `set_axis_bgcolor`, which is only available for older matplotlib versions, but was replaced by `set_facecolor` in newer matplotlib versions. Now, the metawrap installation enforces the older matplotlib version, so its likely that you have a newer version installed somewhere that Python is importing instead. Both the reassemble_bins and bin_refinement module plotting script use the same command, so I assume you must have run them in the slightly different contexts. 

So I would re-try with a new conda environment. You dont have to re-run the module, just run the `~/miniconda2/bin/metawrap-scripts/plot_reassembly.py  *.stats` on the output files to make the plot. If that doesnt work, you can always just comment out the `set_axis_bgcolor` options or change them to `set_facecolor` in the plotting script itself.
